test_runner/src/main/kotlin/ftl/shard/Shard.kt

```diff
@@ -35,6 +36,16 @@ class com.foo.ClassName#testMethodToSkip
 
 */
 
+fun createShardsByTestForShards(args: AndroidArgs) = args.testTargetsForShard.map { list ->
```

Comment:
IMO the extension function will fit better.

Reply:
👍

Reply:
I think my comment https://github.com/Flank/flank/pull/1299#discussion_r526025879 is also applicable in this case